generate-stackbrew-library.sh

```diff
@@ -10,7 +10,7 @@ array_4_8='4 argon';
 # shellcheck disable=SC2034
 array_6_11='6 boron';
 # shellcheck disable=SC2034
-array_8_8='8 latest';
+array_8_9='8 latest';
```

Comment:
I have an open PR to add the "carbon" tag: https://github.com/nodejs/docker-node/pull/562

If you add it here I'll close my PR.

Reply:
Let's merge yours and update this one.